docs/api-docs/analytics/BODL.md

```diff
@@ -0,0 +1,432 @@
+# Big Open Data Layer (BODL)
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Overview](#overview)
+- [Pixel example snippets](#pixel-example-snippets)
+</div> 
+
+## Overview
+The Big Open Data Layer (shortened as BODL, pronounced 'Bottle') is a standard used to obtain data points needed for driving storefront analytics and personalization. Having all BigCommerce and headless storefronts use this standard over time allows merchants to benefit from increased site speed and more consistent third-party analytics integrations.
+
+## Pixel example snippets
+Below is a series of example snippets of code to build on when using BODL data for your integration.
```

Comment:
@markcmurphy, Nate said the following:
it's a generic term / tech for how you track users. here's a good overview: https://www.cookiepro.com/knowledge/tracking-pixel/

it could be seen as 'example analytics snippets' instead?
or 'example tracking snippets'?

Reply:
I recommend "scripts" or something along those lines instead of "pixel"